tldraw/tldraw
# Fix selection foreground being misaligned

Nasty one!

This PR fixes the selection foreground of shapes getting misaligned when the browser zoom was set to something other than 100%. It was always happening all the time on android.

![image](https://github.com/tldraw/tldraw/assets/15892272/f75c19f9-769b-492c-bacd-fe96c5b596ed)

### Change Type


### Test Plan

1. Make a shape.
2. Select it.
3. Change your browser's zoom level.
4. Make sure the selection foreground stay in the right place (eg: indicator, resize handles).


### Release Notes

- None (fix for a bug that hasn't released)
